### PR TITLE
Create sensor for Phicomm M1 air sensor

### PR DESCRIPTION
**Pull request by Yonsm**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13781

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.phicomm.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Phicomm M1 air sensor"
+description: "Instructions on how to integrate Phicomm M1 air sensor within Home Assistant."
+date: 2018-04-19 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: https://cdn-mall.phicomm.com/themes/pc/feixun/images/logo.png
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: Please, do not use external image sources.
Add the logo to the repository and make a local reference.

File: source/_components/sensor.phicomm.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ha_category: Sensor
+ha_release: 0.67
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: `0.67` is already shipped, maybe up it to `0.68`.

File: source/_components/sensor.phicomm.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+sensors:
+  - platform: phicomm
+    name: Phicomm
+    username: ********
+    password: ********
+    devices: 1
+    sensors: [pm25, hcho, temperature, humidity]
+```
+
+Configuration options:
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.phicomm.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The `phicomm` sensor platform let you monitor data from an Phicomm M1 air sensor.
+
+To enable it, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensors:
+  - platform: phicomm
+    name: Phicomm
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.phicomm.markdown [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    username: ********
+    password: ********
+    devices: 1
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: Same as above, optional.

File: source/_components/sensor.phicomm.markdown [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    sensors: [pm25, hcho, temperature, humidity]
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: Same as above, optional.

File: source/_components/sensor.phicomm.markdown [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+sharing: true
+footer: true
+logo: https://cdn-mall.phicomm.com/themes/pc/feixun/images/logo.png
+ha_category: Sensor
+ha_release: 0.67
+ha_iot_class: "Local Polling"
+---
+
+
+The `phicomm` sensor platform let you monitor data from an Phicomm M1 air sensor.
```
Comment (frenck) [inline source/_components/sensor.phicomm.markdown]: - `let` -> `lets`
- `an Phicomm` -> `a Phicomm`

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@Yonsm I've reviewed your PR and left you some comments.
Could you please take a look? 👍

File: source/_components/sensor.phicomm.markdown (hunk 1 above)
Comment (Yonsm) [reply source/_components/sensor.phicomm.markdown]: done

File: source/_components/sensor.phicomm.markdown (hunk 2 above)
Comment (Yonsm) [reply source/_components/sensor.phicomm.markdown]: done

File: source/_components/sensor.phicomm.markdown (hunk 3 above)
Comment (Yonsm) [reply source/_components/sensor.phicomm.markdown]: Please have a look on it. Maybe done?

**Comment by Yonsm** [comment]
@frenck I update the document. Please have a look, thank you.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
@Yonsm Sorry for the delay. I've done a minor tweak and approved it.
Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Parent PR was closed. Closing this one as well.
Feel free to re-open when ready.

**Pull request closed without merging**